icu4c/source/test/cintltst/cloctst.c

```diff
@@ -6018,6 +6019,47 @@ static void TestToLanguageTag(void) {
     }
 }
 
+static void TestBug20132(void) {
+    char langtag[256];
+    UErrorCode status;
+    int32_t len;
+
+    static const char inloc[] = "C";
+    static const char expected[] = "en-US-u-va-posix";
+    static const int32_t expected_len = sizeof expected - 1;
```

Comment:
Please use strlen(expected) not sizeof.

Reply:
Done. But why?

Reply:
MSVC failed compiling that code with the following error message:

```
icu4c\source\test\cintltst\cloctst.c(6029,0): Error C2099: initializer is not a constant
```

So I've now reverted that change.

Reply:
I think strlen() is better because the char[] may or may not be NUL-terminated, and so the -1 may be wrong. sizeof-1 is also less intuitive for the length of a string.

you can get around the compiler error by making the expected_len not "static const".

Reply:
Done.